packages/runtime/src/next/components/tests/controls/__snapshots__/list-control.test.tsx.snap

```diff
@@ -117,7 +117,7 @@ exports[`Page List of colors when value is {"value":[{"swatchId":"[swatch-test-i
               "props": {
                 "propKey": [
                   {
-                    "id": "xxxxxxxx-xxxx-xxxx-xxxx-100000000006",
+                    "id": "xxxxxxxx-xxxx-xxxx-xxxx-100000000003",
```

Comment:
Here and below, update test snapshots to reflect three fewer global `uuid` calls.